packages/rest/src/validation/request-body.validator.ts

```diff
@@ -79,11 +72,29 @@ function convertToJsonSchema(openapiSchema: SchemaObject) {
   return jsonSchema;
 }
 
-// tslint:disable-next-line:no-any
-function validateValueAgainstJsonSchema(body: any, schema: any) {
-  const ajv = new AJV({allErrors: true});
+/**
+ * Validate the request body data against JSON schema.
+ * @param body The request body data.
+ * @param schema The JSON schema used to perform the validation.
+ * @param globalSchemas Schema references.
+ */
+function validateValueAgainstJsonSchema(
+  // tslint:disable-next-line:no-any
+  body: any,
+  // tslint:disable-next-line:no-any
+  schema: AnyObject,
```

Comment:
Should we call this argument `jsonSchema` to make it more clear it is not OpenAPI schema?